src/components/Images.vue

```diff
@@ -299,24 +299,24 @@ export default {
       this.currentCommand = `delete ${ obj.imageName }:${ obj.tag }`;
       this.mainWindowScroll = this.$refs.fullWindow.parentElement.parentElement.scrollTop;
       this.startRunningCommand('delete');
-      ipcRenderer.send('confirm-do-image-deletion', obj.imageName.trim(), obj.imageID.trim());
+      ipcRenderer.send('confirm-do-image-deletion', `${ obj.imageName.trim() }`, `${ obj.imageID.trim() }`);
```

Comment:
How did this get past lint?  Don't we prefer `expn` to `<backquote>${ expn }<backquote>`?   (`<backquote>` because backslash-backquote gave a literal backslash).